src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -176,61 +207,214 @@ def __blob_is_not_complete(metadata):
     return True
 
 
-def acr_build_queue(cmd,
+def acr_queue(cmd,
```

Comment:
>acr_queue [](start = 4, length = 9)

Do we need to prefix this with acr or can we just call it queue_build?

Reply:
The command will be moved to acr command group once we integrate the official sdk.

Reply:
I don't think we have to use the prefix, maybe they were sticklers for having some prefix in the past. I see a lot of teams that don't follow the convention, e.g., cosmosdb. At the very least for now it keeps us consistent with the rest of the work @djyou has done and the only one it affects is the one reading the source code of the CLI.

Reply:
It should actually be `acr_build` as the command is `acr build`. Simply way just replace space with underscore from command as method name.